### PR TITLE
Eth2 scams page [proof-of-concept]

### PR DESCRIPTION
**Pull request by ryancreatescopy**
## Description

A place to warn users against known scams and provide users a simple way to report scams they've come across – to make staking a safer place. Just a proof-of-concept at this stage.

## Related Issue

None yet.

### CODE REVIEW

**Comment by ryancreatescopy** [comment]
Rather vitally, I can't get the scam image to render just yet...

- [ ] Fix images

File: src/components/ScamCard.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+          title,
+          description,
+          image,
+          children,
+          scamUrl,
+          date,
+          platform,
+        } = listItem
+        return (
+          <CardContainer id={idx}>
```
Comment (wackerow) [inline src/components/ScamCard.js:78]: changing to `key={idx}` removes a console warning

File: src/pages/eth2/scams.js [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+  justify-content: center;
+  margin-right: 0.5rem;
+`
+
+const GithubIcon = styled(Icon)`
+  fill: ${(props) => props.theme.colors.text};
+  margin-right: 0.5rem;
+`
+
+const KnownScamsPage = (data) => {
```
Comment (wackerow) [inline src/pages/eth2/scams.js:97]: destructure `data` using `({ data })`

File: src/pages/eth2/scams.js [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+      scamUrl: "https://v2.stake-ethereum.org/?ref=adsjoe7",
+      /* image: data.discordscam.childImageSharp.fixed, */
+      date: "11/05/2020",
+      platform: "discord",
+    },
+    {
+      title: "Discord scam",
+      description: "test description",
+      scamUrl: "https://v2.stake-ethereum.org/?ref=adsjoe7",
+      /* image: data.discord.childImageSharp.fluid, */
```
Comment (wackerow) [inline src/pages/eth2/scams.js:112]: change back to `.fixed`

File: src/pages/eth2/scams.js [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+  query {
+    discordscam: file(relativePath: { eq: "eth2-scams/discord-scam.png" }) {
+      childImageSharp {
+        fixed(width: 400) {
+          ...GatsbyImageSharpFixed
+        }
+      }
+    }
+  }
+`
```
Comment (wackerow) [inline src/pages/eth2/scams.js:207]: image

Those changes allow the image to render on my end.

Comment (wackerow) [reply src/pages/eth2/scams.js:207]: (oops, not sure why this got tagged to the query lines... this query worked for me after above fixes)

**Comment by samajammin** [comment]
@ryancreatescopy closing this out for the time being. I'll keep the branch intact so we can return to this if wanted.

**Pull request closed without merging**